src/website/views/components/marketing-icon/_examples.pug

```diff
@@ -1,3 +1,10 @@
+- var iconsPlatform = ['platform-adaptive-networking', 'platform-connected-security', 'platform-edge-compute', 'platform-it-agility-hybrid-cloud', 'platform-voice-unified-communications'];
+- var iconsProducts = ['products-apps', 'products-agility','products-ai-machine-learning','products-analytics','products-analytics-data-dashboards','products-apps-other','products-big-data-apps-and-services','products-bug','products-c2-command-and-control-server','products-cce-dynamic-connections','products-cdn-service','products-cisco-unified-comm-manager','products-cloud-saas-applications','products-cloud-solutions','products-cloud-storage','products-collaboration','products-conferencing','products-custom-backgrounds','products-data','products-database', 'products-designs','products-ddos','products-edge-security','products-efficiency','products-email','products-email-security','products-end-user-remote','products-energy-and-utilities','products-ethernet','products-file-sharing','products-financial-services','products-financial-services-company','products-firewall','products-flexibility','products-healthcare','products-hyperscalers','products-infected','products-infected-server','products-internet','products-internet-cloud','products-iot','products-load-balancer','products-lumen-adaptive-virtual-services','products-lumen-edge-orchestrator','products-malware','products-managed-services','products-manufacturing','products-media-and-entertainment','products-migration','products-mpls-ip-vpn-service','products-news','products-online-shopping','products-pharma','products-plane-satellite-internet','products-portal','products-private-branch-exchange','products-private-fiber-network','products-retail','products-route-miles','products-router','products-sap-applications','products-satellite','products-screen-sharing','products-sd-wan','products-sd-wan-router-intelligent-routing','products-security','products-security-endpoint-application','products-services','products-session-border-controllers','products-ship-satellite-internet','products-sip-voice-calling','products-speed','products-system-integrators','products-tech-support','products-teleport','products-threat-detect','products-threat-intelligence','products-total-solution','products-train-satellite-internet','products-uc-and-c','products-ui-ux-modernization','products-upload-download','products-video-chat','products-virtual-desktop','products-voice','products-vpn-gateway','products-vulnerability','products-warning','products-wavelength','products-web-application-firewall','products-wifi', 'products-network-solutions'];
```

Comment:
I will also include a fix for this - `products-network-solutions` isn't ordered alphabetically